keylime/cloud_verifier_tornado.py

```diff
@@ -848,6 +848,8 @@ def post(self):
             logger.warning("POST returning 400 response. Expected non zero content length.")
             return
 
+        ima_policy = json.dumps("{}")
```

Comment:
I don't think this is right.
```
>>> from keylime import json
>>> json.dumps("{}")
'"{}"'
>>> v=json.dumps("{}")
>>> v
'"{}"'
```

  Maybe it should be `json.loads("{}")` but why go via json?

Reply:
a bizarre mistake, evidently... should be simply a string